<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="bLoad">
      <property name="text">
       <string>Загрузить Длл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bFree">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Выгрузить Длл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bWork">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Дернуть метод</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
